terraform/modules/enclave/verify-dns/main.tf

```diff
@@ -0,0 +1,14 @@
+data "aws_route53_zone" "private_hosted_zone" {
+  name   = "${var.hosted_zone_name}"
+  vpc_id = "${var.target_vpc}"
+}
+
+resource "aws_route53_record" "prometheus" {
+  count = "2"
```

Comment:
Am I right in thinking we could remove the hardcoding of 2 here and use the length of `var.prometheus_private_ips`?